.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: node_js
 
 node_js:
-  - "7"
+  - "8"
```

Comment:
On the main node site, they list v6 as the stable LTS version and v8 as the current (latest). 

Think we should only track and support those two versions in our builds.